manuscripts/report.py

```diff
@@ -134,11 +134,19 @@ def __init__(self, es_url, start, end, data_dir=None, filters=None,
             self.data_sources.remove('github')
             self.data_sources += ['github_issues', 'github_prs']
         self.data_sources = list(set(self.data_sources))
+        # Check for custom indices
+        self.indices = indices if indices else []
         # End temporal hack
         self.config = self.__get_config(self.data_sources)
         self.report_name = report_name
         self.projects = projects
 
+        # Create an index name dict which, for each data_source, will give the
+        # name of the elasticsearch index that has to be used.
+        self.index_dict = defaultdict(lambda: None)
+        for i in range(len(self.indices)):
```

Comment:
What about `for pos, index in enumerate(self.indices)` ? I find it more pythonic, and it gives you index, that you need below, in the body of the loop.